plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositorySpoonStartExtensionPoint.java

```diff
@@ -33,8 +33,7 @@
 import org.pentaho.di.pan.CommandLineOption;
 import org.pentaho.di.repository.RepositoryMeta;
 import org.pentaho.di.ui.repo.controller.RepositoryConnectController;
-import org.pentaho.di.ui.repo.dialog.RepositoryConnectionSWT;
```

Comment:
Copyright date on this file needs to be updated.

Reply:
fixed